### PR TITLE
New package: Beyond-All-Reason.Beyond-All-Reason version 1.2988.0

### PR DESCRIPTION
**Pull request by wvanderp**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/155045)

### CODE REVIEW

**Review by wvanderp — COMMENTED** [review]
updated to make it Silent

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 881.)

File: manifests/b/Beyond-All-Reason/Beyond-All-Reason/1.2988.0/Beyond-All-Reason.Beyond-All-Reason.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Beyond-All-Reason.Beyond-All-Reason
+PackageVersion: 1.2988.0
+InstallerType: nullsoft
+InstallerSwitches:
+  Silent: /S
+Installers:
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/b/Beyond-All-Reason/Beyond-All-Reason/1.2988.0/Beyond-All-Reason.Beyond-All-Reason.installer.yaml:10]:
```suggestion
- Architecture: x64
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @wvanderp,

Our `x86` validations are performed on 32-bit OSes. This appears to be a 64-bit application.

File: manifests/b/Beyond-All-Reason/Beyond-All-Reason/1.2988.0/Beyond-All-Reason.Beyond-All-Reason.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Beyond-All-Reason.Beyond-All-Reason
+PackageVersion: 1.2988.0
+InstallerType: nullsoft
+InstallerSwitches:
+  Silent: /S
```
Comment (Trenly) [inline manifests/b/Beyond-All-Reason/Beyond-All-Reason/1.2988.0/Beyond-All-Reason.Beyond-All-Reason.installer.yaml:8]:
```suggestion
```
This shouldn't be necessary, as it's the default for nullsoft installer types

File: manifests/b/Beyond-All-Reason/Beyond-All-Reason/1.2988.0/Beyond-All-Reason.Beyond-All-Reason.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Beyond-All-Reason.Beyond-All-Reason
+PackageVersion: 1.2988.0
+PackageLocale: en-US
+Publisher: Beyond All Reason
```
Comment (Trenly) [inline manifests/b/Beyond-All-Reason/Beyond-All-Reason/1.2988.0/Beyond-All-Reason.Beyond-All-Reason.locale.en-US.yaml:7]:
```suggestion
Publisher: BAR Team
```
This should match control panel

File: manifests/b/Beyond-All-Reason/Beyond-All-Reason/1.2988.0/Beyond-All-Reason.Beyond-All-Reason.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Beyond-All-Reason.Beyond-All-Reason
+PackageVersion: 1.2988.0
+PackageLocale: en-US
+Publisher: Beyond All Reason
+PackageName: Beyond-All-Reason
+License: copyrighted
```
Comment (Trenly) [inline manifests/b/Beyond-All-Reason/Beyond-All-Reason/1.2988.0/Beyond-All-Reason.Beyond-All-Reason.locale.en-US.yaml:9]: A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/b/Beyond-All-Reason/Beyond-All-Reason/1.2988.0/Beyond-All-Reason.Beyond-All-Reason.locale.en-US.yaml (hunk 4 above)
Comment (wvanderp) [reply manifests/b/Beyond-All-Reason/Beyond-All-Reason/1.2988.0/Beyond-All-Reason.Beyond-All-Reason.locale.en-US.yaml:9]: I could help you figure out this.

Some parts are as open as MIT or cc-by. Others are proprietary/copyrighted.

So overall I think that this whole package should be `Proprietary`.

File: manifests/b/Beyond-All-Reason/Beyond-All-Reason/1.2988.0/Beyond-All-Reason.Beyond-All-Reason.locale.en-US.yaml (hunk 3 above)
Comment (wvanderp) [reply manifests/b/Beyond-All-Reason/Beyond-All-Reason/1.2988.0/Beyond-All-Reason.Beyond-All-Reason.locale.en-US.yaml]: Should it only match here or should I change all the instances of this publisher?

File: manifests/b/Beyond-All-Reason/Beyond-All-Reason/1.2988.0/Beyond-All-Reason.Beyond-All-Reason.locale.en-US.yaml (hunk 4 above)
Comment (stephengillie) [reply manifests/b/Beyond-All-Reason/Beyond-All-Reason/1.2988.0/Beyond-All-Reason.Beyond-All-Reason.locale.en-US.yaml:9]: I looked through the project repo and website, and couldn't find licensing details beyond "free to download and install". Unfortunately, no `License.txt` was present.

File: manifests/b/Beyond-All-Reason/Beyond-All-Reason/1.2988.0/Beyond-All-Reason.Beyond-All-Reason.locale.en-US.yaml (hunk 3 above)
Comment (stephengillie) [reply manifests/b/Beyond-All-Reason/Beyond-All-Reason/1.2988.0/Beyond-All-Reason.Beyond-All-Reason.locale.en-US.yaml]: I think it only has to match here, not the Publisher portion of the `PackageIdentifier`.

**Comment by stephengillie** [comment]
Verify:
image

**Review by stephengillie — APPROVED** [review]

**Comment by wvanderp** [comment]
Thank you all for the feedback and a great Process

**Pull request merged**